content/blog/kubecon-2023.md

```diff
@@ -0,0 +1,54 @@
+---
+id: blog
+title: Kubecon, is it still for me?
+description: Lee looks back on his second Kubecon, lists some standout-talks and discusses his takeaway.
+summary: Lee looks back on his second Kubecon, lists some standout-talks and discusses his takeaway.
+author: Lee Beenen
+tags:
+  - cloud
+  - kubernetes
+  - k8s
+  - cloud native security
+  - vulnerability scanning
+slug: kubecon-2023
+date: 2023-05-05
+---
+Kubecon 2023 is a wrap! In fact, it's already a couple of weeks in the past. As only my second Kubecon ever (Kubecon Valencia 2022 was my first), it leaves me with a lot of mixed feelings. 
+
+Yes, it was a well run event, it was filled with interesting talks and like last year, I learned about a lot of stuff happening outside the bubble of my current assignment, which is one of the main attractions of being a consultant at Skyworkz, and going to a tech conference to meet people.
+
+However, KubeCon also left me thinking that I am no longer its target audience. Why? 
+
+We'll get to that, let's first lead off with couple of stand-out talks that I feel are worth a (re)-watch.
```

Comment:
I'd replace the colon with a period. It seems to work better that way for me.

Reply:
Agreed!